### PR TITLE
Fdg 5671

### PR DESCRIPTION
**Pull request by fzafrb**
resolves FDG-5671

https://federal-spending-transparency.atlassian.net/browse/FDG-5671

88.84 test coverage

### CODE REVIEW

**Comment by wyattch** [comment]
This review cannot be completed until 6002 (data variables ticket) is merged.

File: src/layouts/explainer/sections/federal-spending/federal-spending.jsx [hunk 1]
```diff
@@ -1,64 +1,76 @@
...
+      "The Difference Between Mandatory, Discretionary, and Supplemental Spending",
+    component: (glossary, cpiDataByYear) => (
+      <SpendingDifference glossary={glossary} />
+    ),
   },
   {
     index: 4,
     id: nationalDeficitSectionIds[4],
-    title: 'Spending Trends Over Time and the U.S. Economy',
+    title: "Spending Trends Over Time",
```
Comment (j-hertzler) [inline src/layouts/explainer/sections/federal-spending/federal-spending.jsx:68]: Title in the MAC for this section is Spending Trends Over Time and the U.S. Economy

File: src/layouts/explainer/sections/federal-spending/spending-difference/spending-difference.jsx [hunk 2]
```diff
@@ -1,108 +1,173 @@
...
+import GlossaryTerm from "../../../../../components/glossary-term/glossary-term"
+import CustomLink from "../../../../../components/links/custom-link/custom-link"
+import supplementalSpendingDesktop from "../../../../../../static/images/supplemental-spending_desktop.png"
+import supplementalSpendingMobile from "../../../../../../static/images/supplemental-spending_mobile.png"
+import MandatorySpendingImgDesktop from "../../../../../../static/images/mandatory-spending_desktop.png"
+import MandatorySpendingImgMobile from "../../../../../../static/images/mandatory-spending_mobile.png"
 
-export const SpendingDifference = ({width}) => {
+export const SpendingDifference = ({ width, glossary }) => {
+  useEffect(() => {
```
Comment (j-hertzler) [inline src/layouts/explainer/sections/federal-spending/spending-difference/spending-difference.jsx:22]: This useEffect should be removed

File: src/layouts/explainer/sections/federal-spending/spending-categories/spending-categories.jsx [hunk 3]
```diff
@@ -1,55 +1,58 @@
...
-          title="What does the future of Social Security and Medicare look like?"
-        >
-          Each year, the Social Security and Medicare Boards of Trustees publish their Annual
-          Report on the Financial Status of Social Security and Medicare. The Boards’ projections
-          indicate that spending will continue to increase. As the average age of Americans
-          increases, more funding is needed to support entitlement programs like Social Security,
-          Medicare, and retirement and disability services for both military and civil servants.
+        <Accordion title="What does the future of Social Security and Medicare look like?">
+          Each year, the Social Security and Medicare Boards of Trustees publish
+          their
```
Comment (j-hertzler) [inline src/layouts/explainer/sections/federal-spending/spending-categories/spending-categories.jsx:42]: Noticed a space missing before and after the link when running this locally

File: src/layouts/explainer/sections/federal-spending/federal-spending.jsx (hunk 1 above)
Comment (j-hertzler) [reply src/layouts/explainer/sections/federal-spending/federal-spending.jsx:68]: Confirm with communications what the correct title is- The MAC has two versions of this section title that are different, one in the table of contents and one as the section title

Comment (fzafrb) [reply src/layouts/explainer/sections/federal-spending/federal-spending.jsx]: will do @j-hertzler , who is the right person to confirm with?

**Review by j-hertzler — APPROVED** [review]
Changes look good and copy matches the MAC. Tests and build are passing locally

**Pull request merged**